### DISCUSSION TITLE
Why is heading h2 and not h1?

### DISCUSSION THREAD

**Discussion by IanVS** [PaloAltoNetworks/docusaurus-openapi-docs#419] [Q&A]
The styling of the main page heading does not match the rest of my docs because it's using an `h2` instead of the `h1` that docusaurus uses for markdown pages by default.  I think since it's the top-most heading on the page, semantically it should be an `h1` as well, right?

What's the best way to modify that in my own site, if you don't want to make the change globally?

Also, sorry for all the questions and feedback lately.  I like this tool, just trying to get the most out of it.  ;)

**Comment by sserrata** [comment]
Hi @IanVS, I think the rationale is to make it a bit smaller to better support/fit really long summaries/headings. That said, we can consider reverting it back to an`h1` if it makes it more consistent with non-API docs.

**Reply by IanVS** [reply]
I think that would be the semantically-correct thing to do, personally.  AFAIK, each page should have one `h1` heading near the top of the page to describe the overall page.  See https://www.a11yproject.com/posts/how-to-accessible-heading-structure/ for example.

**Reply by sserrata** [reply]
I agree, it should be `h1` for a11y standards. If anything, we can also provide a CSS classname to override the `font-size` if desired.

**Reply by IanVS** [reply]
You could, or it's probably easy enough to target `.theme-api-markdown h1` as well.
